Tests/CommandsTests/PackageCommandTests.swift

```diff
@@ -48,10 +48,9 @@ final class PackageCommandTests: CommandsTestCase {
         XCTAssertMatch(stdout, .contains("USAGE: swift package"))
     }
 
-    func testUsage() async throws {
-        throw XCTSkip("rdar://131126477")
+    func testInvalidUsage() async throws {
         do {
-            _ = try await execute(["-halp"])
+            _ = try await execute(["--xxx"])
```

Comment:
issue: Can we add a test, if they don't currently exist, that specified the behaviour when executing `swift build -halp`, `swift build -alhp` and `swift build --help`

Reply:
I think these tests should probably go in swift-argument-parser, since it handles the help arguments

Reply:
Similar tests should definitely be included in swift-argument-parser. However, from swift-package-manager perspective, it's an implementation details.

This is a `large` (ie: end-to-end) test, where the code under test is not visible. So from the tests's perspective, we don't know we are using swift-argument-parser, or maybe we will use a different argument parser in the future, which breaks the behaviour.  As such, tests should be included in swift-package-manager that verify the expected behaviour as a software developer would use the command line.  We should include `large` (ie: end-to-end) tests that cover the expected behaviour, or maybe at least describe the existing behaviour.